modelzoo/WnD/TensorFlow2/wnd.patch

```diff
@@ -21621,42 +21621,42 @@ index d673f74..78d48ed 100644
 --- a/modelzoo/WnD/TensorFlow2/main.py
```

Comment:
what are the new changes on the patch file? did you modified the patches?

Reply:
yes, I modified patch for WnD to run in docker.